src/exporter.py

```diff
@@ -151,7 +145,11 @@ def track_task_event(self, event):
         if event["type"] not in self.state_counters:
             logger.warning("No counter matches task state='{}'", task.state)
 
-        labels = {"name": task.name, "hostname": get_hostname(task.hostname)}
+        labels = {
+            "name": task.name,
+            "hostname": get_hostname(task.hostname),
+            "queue": getattr(task, "queue", "default"),
```

Comment:
The default queue is "celery"

https://docs.celeryq.dev/en/stable/userguide/configuration.html#task-default-queue